codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -865,7 +865,7 @@ impl Default for SyncConfig {
         Self {
             initial_block_interval: 10_000,
             backoff_multiplicative: 0.8,
-            acceleration_additive: 2_000,
+            acceleration_additive: 500,
```

Comment:
Changed after discussion

Reply:
We must still monitor if this is low enough I think. If queries are maxing out below 2500 here then backoff will be less than the acceleration. Obviously it's configurable so it can be dealt with case by case.